src/Reshuffle.ts

```diff
@@ -123,7 +124,7 @@ export default class Reshuffle {
     })
   }
 
-  handleEvent(eventName: string, event: any): boolean {
+  async handleEvent(eventName: string, event: any): Promise<boolean> {
```

Comment:
Not sure what happened. My code has the async in it (otherwise I couldn't have tested it).

Reply:
I think this could be lint. I see automatic changes in other parts of the code as well (e.g. removed semicolons). Not sure we want to have auto formatting